backend/db/scripts/migrations/01-migration.sql

```diff
@@ -0,0 +1,20 @@
+CREATE DATABASE movie_matcher;
+
+
+CREATE TABLE sessions (
+    id varchar(255) UNIQUE NOT NULL PRIMARY KEY,
```

Comment:
whe can let the db auto create the id for us, unless you need to generate it outside the db.